packages/fabric8-ui/src/app/space/app-launcher/services/app-launcher-dependency-check.service.ts

```diff
@@ -39,9 +35,7 @@ export class AppLauncherDependencyCheckService implements DependencyCheckService
    *
    * @returns Observable
    */
-  getApplicationsInASpace(): Observable<Application[]> {
-    return this.deploymentApiService.getApplications(
-      this.context.space ? this.context.space.id : '',
-    );
+  getApplicationsInASpace(): Observable<any[]> {
+    return of([])
```

Comment:
@vikram-raj how is this used in launcher?

Reply:
@christianvogt It is use to validate the application name while creating the application. So, that user can not create multiple application with the same name.

Reply:
@corinnekrych can we use the same workaround here to get the application names?

Reply:
@corinnekrych ```getApplicationsInSpace``` is getting called in ```project-name.validator``` directive in ngx-launcher. https://github.com/fabric8-launcher/ngx-launcher/blob/master/projects/ngx-launcher/src/lib/shared/project-name.validator.ts#L34

Reply:
@edewit do you know why we used project-name.validator given that we've already checked the name is not already in used in https://github.com/fabric8-ui/fabric8-ui/blob/8ebda0a75136aaaa525d54da5c018e026ce01409/packages/fabric8-ui/src/app/space/add-app-overlay/add-app-overlay.component.ts#L165 ?

Reply:
@corinnekrych Looks like the app launcher has two more areas after the initial screen where the user can change the application name:

![image](https://user-images.githubusercontent.com/14068621/51919281-192bdf80-23b1-11e9-8a88-a7ab655f333e.png)
![image](https://user-images.githubusercontent.com/14068621/51919291-1d57fd00-23b1-11e9-9ff9-a11ad8b47029.png)

Reply:
gotcha indeed!